objectsapiclient/models.py

```diff
@@ -93,3 +93,51 @@ def get_choices(
                     choices = blank_choice + choices
 
         return choices
+
+
+class LazyObjectTypeField(ObjectTypeField):
+    """
+    Custom `ObjectTypeField` that fetches objecttype choices only if:
+        1. The database table exists (migrations have been run)
+        2. The Objects API services are actually configured
+    This prevents:
+        - Unnecessary HTTP requests at server startup
+        - Database errors when migrations haven't been run yet
+    """
+
+    def get_choices(
+        self,
+        include_blank=True,
+        blank_choice=BLANK_CHOICE_DASH,
+        limit_choices_to=None,
+        ordering=(),
+    ):
```

Comment:
Let's beware of mutable defaults here:

```suggestion
    def get_choices(
        self,
        include_blank=True,
        blank_choice=None,
        limit_choices_to=None,
        ordering=None,
    ):
```

And replace refs below with `symbol or default`

Reply:
Why diverge from the parent `ObjectTypeField` on `blank_choice=BLANK_CHOICE_DASH` (I copied from there)?

Reply:
Boyscout rule I suppose, leave things slightly beter than we found them (but we'd have to do that in the parent as well).

But agreed, let's leave this for now and defer this to a future `ruff` upgrade, which will catch it.